modules/data.atmosphere/vignettes/ameriflux_demo.Rmd

```diff
@@ -11,9 +11,11 @@ vignette: >
 
 # Overview
 
-This is a demonstration of the PEcAn utilities for downloading met data, converting it to the PEcAn-CF format (which is based on the Climate Forecasting conventions and similar to MsTMIP). These variables are defined in the [PEcAn documentation](https://pecanproject.github.io/pecan-documentation/latest/met-data.html).
+This is a demonstration of the PEcAn utilities for downloading met data, converting it to the PEcAn-CF format (which is based on the Climate Forecasting conventions and similar to MsTMIP). These variables are described in the [PEcAn met data documentation](https://pecanproject.github.io/pecan-documentation/) (link previously pointed to a broken page).
+
+We’ll download 12 years of met data from the [Bondville Ameriflux site](https://ameriflux.lbl.gov/sites/siteinfo/US-Bo1), which has a `SITE_ID` of `US-Bo1`.
+<!-- Old link: http://ameriflux.ornl.gov/fullsiteinfo.php?sid=44 -->
```

Comment:
Thanks for finding the new link! OK to drop the old one completely with no comment.
```suggestion
We’ll download 12 years of met data from the [Bondville Ameriflux site](https://ameriflux.lbl.gov/sites/siteinfo/US-Bo1), which has a `SITE_ID` of `US-Bo1`.
```